src/trampoline.rs

```diff
@@ -59,9 +59,9 @@ impl ThreadsafeTrampoline {
     /// Unreferences a trampoline to allow exiting the event loop before it has been dropped.
     /// Safety: `Env` must be valid for the current thread
     /// _Not idempotent_
-    pub(crate) fn unref(&mut self, env: Env) {
+    pub(crate) fn decrement_references(&mut self, env: Env) {
         if self.ref_count == 0 {
-            return;
+            panic!("ThreadsafeTrampoline reference underflow");
         }
```

Comment:
Ah, Rust tip: this is probably better written as `assert!(self.ref_count > 0, "…")`, which is exactly equivalent.

Reply:
Good point. Fixed!